### PR TITLE
Upgrade to v1.45.3

### PR DESCRIPTION
**Pull request by chalenge**
https://github.com/faros-ai/metabase/releases/tag/v1.45.3

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -2,13 +2,18 @@
 # STAGE 1: builder
 ###################
 
-FROM metabase/ci:java-11-clj-1.11.0.1100.04-2022-build as builder
+FROM node:16-slim as builder
```
Comment (ogusak) [inline Dockerfile:5]: @chalenge - we 'd need to update our build as well

Comment (ogusak) [reply Dockerfile:5]: we'd need to do it before merging this - so when the build pipeline is triggered it would pick up the new changes in the Dockerfile

**Review by ogusak — APPROVED** [review]

File: Dockerfile (hunk 1 above)
Comment (chalenge) [reply Dockerfile:5]: yeah, good catch

Comment (chalenge) [reply Dockerfile:5]: @ogusak [confirm this is the update we needed](https://github.com/faros-ai/metabase-container/pull/111/files#diff-b813a858c0796a56d0f0c00a13d0455951d811f61c74fbd5b6dea27816dbabc5R13)

Comment (ogusak) [reply Dockerfile:5]: also this one https://github.com/faros-ai/metabase-container/blob/main/Dockerfile#L1 - to pick the new image base

**Pull request merged**